CMakeLists.txt

```diff
@@ -127,11 +144,19 @@ if (CONDA_BUILD)
         set (MAP_IMAGES_PATH $ENV{MAP_IMAGES_PATH})
     endif()
 else()
+    message("THIRD_PARTY_DIR = ${THIRD_PARTY_DIR}")
     include_directories (SYSTEM ${THIRD_PARTY_INC_DIR})
     include_directories (SYSTEM ${THIRD_PARTY_INC_DIR}/freetype2)
     link_directories (${THIRD_PARTY_LIB_DIR})
     link_directories (${PYTHONPATH})
+
     list (APPEND CMAKE_PREFIX_PATH ${THIRD_PARTY_LIB_DIR})
+    list (APPEND CMAKE_PREFIX_PATH ${THIRD_PARTY_DIR})
+    list (APPEND CMAKE_PREFIX_PATH ${THIRD_PARTY_DIR}/HDF_Group/HDF5/1.12.2/lib)
+
+    if (APPLE)
+        set(CMAKE_MODULE_PATH /opt/local/lib/libomp)
```

Comment:
This seems like a non-standard path that is not part of the third party libs?

Reply:
Yes, we are not building libomp so it's not in ${THIRD_PARTY_LIB_DIR}.  This is where macports installs it.

Reply:
Can we just install it with the rest of our third party libs?

Reply:
It's not something that I've explored.  In the past, we've installed OMP through clang13+ macports, and relied on macports' default installation path.

Reply:
It seems that at least one other established software (Matlab) encourages users to download the source code and compile OpenMP as a means of obtaining OpenMP support. (https://www.mathworks.com/help/coder/ug/install-openmp-library-on-macos-platform.html) Given that we've spent so much effort putting together a package of 3rd party libraries, I think it'd be a very good idea to complete it with OpenMP.